api.raml

```diff
@@ -4,14 +4,16 @@ version: 4
 protocols: [ HTTPS ]
 mediaType: application/json
 securitySchemes:
-  AuthzBearerToken:
-      type: x-{other}
+ AuthzHttpOnlyCookie:
```

Comment:
Indentation changed on this line. Might be because this section was indented strange. Could you take a look and see what needs to be done to make it consistent with the rest of the file?